iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Request/AppUserRequest.java

```diff
@@ -21,7 +21,7 @@
     private String lastName;
 
     @NotBlank(message = "The username of the user is not valid!")
-    @Size(min = 5)
+    @Size(min = 5, message = "The username length should be greater than 5")
```

Comment:
Isn't min inclusive? Meaning that the username should be greater than or equal to 5? Same applies to the resto of the messages